app/config/platforms.php

```diff
@@ -28,7 +28,7 @@
                 'gitUrl' => 'git@github.com:appwrite/sdk-for-web.git',
                 'gitRepoName' => 'sdk-for-web',
                 'gitUserName' => 'appwrite',
-                'gitBranch' => 'master',
+                'gitBranch' => 'dev',
```

Comment:
I don't see a reason for deleting the `dev` branch. 

If our sdk repositories were conventional repositories and the code not generated by an `sdk-generator`, it would follow a similar development cycle wherein changes are pushed to different feature branches, PRs created, review comments addressed and commits pushed to the same branch which is eventually merged to master/ main. We would be following a similar approach.

If you think its still an issue, it can be addressed by [squashing all the commits to a single commit](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges#squash-and-merge-your-commits) while merging.

Reply:
I love clean commits so I like the idea of squashing 👍🏼